src/App.js

```diff
@@ -72,23 +73,23 @@ class App extends React.Component {
     }
   };
 
-  recordLastNum = (result) => {
-    const regex = /[-+/*]{0,}[0-9]{1,}[.]{0,1}[0-9]*/g;
-    let results = result.match(regex) || [];
-    if (results.length === 0) {
-      return "";
-    }
-    //We need to take the last one
-    let lastNum = results[results.length - 1];
-    // If there is no operator -> we return the value
-    if (!"-+/*".includes(lastNum[0])) {
-      return lastNum;
-    }
-    this.setState({
-      operate: lastNum[0],
-    });
-    return lastNum.substr(1);
-  };
+  // recordLastNum = (result) => {
```

Comment:
@grimxreaper Be careful not to commit commented code. Either by making a habit to delete the code or a better one would be checking the diff before a commit. May help writing the commit message.

Reply:
That was already there I believe...I am not deleting comments that were previously there, thought that's what you would also want...no?

Reply:
I will just delete them from now on. 👍🏼

Reply:
The function was there, yes. But it was commented out and then committed as a comment. That's what this diff is telling me. The next commit deletes this commented function, which is good, but it's better not to commit commented code.

I agree, better to delete rather than commenting. You've got the code in source if you commit regularly and need to get back to it.